collector/config.yaml

```diff
@@ -58,4 +58,9 @@ service:
       exporters: [otlp,debug]
   telemetry:
     metrics:
-      address: localhost:8888
+      readers:
+        - pull:
```

Comment:
Ah ok. So this simply lines up with the upstream PR you linked. Also noting that no changes are required for [end usage docs](https://documentation.solarwinds.com/en/success_center/observability/content/intro/services/aws-lambda-overview.htm).